src/components/Settings/Settings.js

```diff
@@ -30,18 +42,122 @@ export default class Settings extends React.Component {
             <option value="TestNet">TestNet</option>
             <option value="CozNet">CozNet</option>
             <option value="nOSLocal">nOS Local</option>
+
+            {
+              this.props.allNetworks.map((network) => {
+                return (
+                  <option
+                    key={network.neoscan}
+                    value={network.neoscan}
+                  >
+                    {network.name}
+                  </option>
+                );
+              })
+            }
           </select>
         </label>
+        <div className={styles.buttonContainer}>
+          Custom Neoscan Url: {this.props.currentNetwork}
+        </div>
+        {this.renderButtons()}
         {this.state.saved && <Saved /> }
       </div>
     );
   }
 
+  renderButtons = () => {
+    return (
+      <div>
+        <Button onClick={this.handleAddNewNetwork}>
+          {'Add custom network configuration'}
+        </Button>
+        <div className={styles.divider} />
+        <Button onClick={this.handleClearNetwork}>
+          {'Clear custom network configurations'}
+        </Button>
+      </div>
+    );
+  }
+
+  handleClearNetwork = () => {
+    this.props.clearNetworks();
+    this.props.setCurrentNetwork('TestNet');
```

Comment:
Do we always want to switch to TestNet when the networks are cleared?  It seems like we should only do that if the current network is one that's being cleared.